pkg/vm/slot.go

```diff
@@ -47,7 +47,7 @@ func (s *Slot) Set(i int, item stackitem.Item) {
 // Get returns item contained in i-th slot.
 func (s *Slot) Get(i int) stackitem.Item {
 	if item := s.storage[i]; item != nil {
-		return item
+		return item.Dup()
```

Comment:
If we assume that any stack item can be referenced multiple times then we don't need this.